### PR TITLE
feat (helpers): add strings helper

### PR DESCRIPTION
**Pull request by proullon**

### CODE REVIEW

File: helpers/helpers.go [hunk 1]
```diff
@@ -78,6 +78,12 @@ var ProtoHelpersFuncMap = template.FuncMap{
 	"kebabCase": func(s string) string {
 		return strings.Replace(xstrings.ToSnakeCase(s), "_", "-", -1)
 	},
+	"contains": func(sub, s string) bool {
+		return strings.Contains(s, sub)
+	},
+	"trimstr": func(cutset, s string) string {
```
Comment (jhayotte) [inline helpers/helpers.go]: I would not specify `str` and name it directly `trim`

Comment (proullon) [inline helpers/helpers.go]: trim is already a builtin function that trim spaces in templates, there is name collision. So yeah not ideal but needed

Comment (jhayotte) [inline helpers/helpers.go]: ok understood

**Comment by moul** [comment]
👍

**Pull request merged**